src/pk_compiler.c

```diff
@@ -129,7 +129,7 @@ typedef enum {
   TK_WHILE,      // while
   TK_FOR,        // for
   TK_IF,         // if
-  TK_ELIF,       // elif
+  TK_ELSEIF,     // else if
```

Comment:
It should be `TK_ELSIF` and the keyword we're trying to rename to is `elsif`

Reply:
I misunderstood the requirement, my bad. Changed the else if notation to elsif.